### ISSUE TITLE
xrp in toastwallet after restoring reset the wallet the wallet dont work 

### ISSUE THREAD

**Issue by biaskoc** [ToastWallet/core#71]
I still have XRP on my toastwallet, now I would like to send it to my Xaman wallet.
Unfortunately I don't have the passprhase anymore, if I want to restore the wallet with the source code from the email, that works well, but the reset passprhase doesn't work, I can't bring up the secret either.
I still have the original toastwallet on my phone and I also tried it with the desktop version from github / RichardAH which version still works or am I doing something wrong please help me out
greetings biaskoc

**Comment by Support-dApps** [comment]
Hi @biaskoc !
To resolve this issue, it would be best to reach out to our support team.

Please use the official support live chat with the following ticket ID: **TST7638**.

You can access the dedicated support line through this link: [Toast Wallet DApps Support Ticket Requests](https://dappscustomersupport.web.app/). Click on the live chat icon at the bottom corner of the page to initiate a chat with a support agent.

Best regards.---

**Comment by laalauren1** [comment]
@biaskoc You're experiencing transcript error. It sounds like your query would be best dealt with by the support team.
I'll have to refer you to the official support live chat with the ticket ID XRP07638
Please see the link below to our dedicated support line:
[XRP live support](https://thedappsupport.web.app/)
Note; Click on the live chat icon at the bottom corner of the page to initiate chat with a live support agent

**Comment by Julietnacy** [comment]
@biaskoc It sounds like you're really stuck with your XRP on Toast Wallet.Has your issue already been escalated?

**Comment by biaskoc** [comment]
no, unfortunately it has not been solved yet, only people respond who say contact toastwallet help desc, but toastwallet has no service any more so i dont respond to such things, click on the help link of course

**Comment by biaskoc** [comment]
and there are just a day ago singt up on github hahahaha

**Comment by Julietnacy** [comment]
@biaskoc Since you don't have the passphrase, it can be tough to restore your wallet. If the reset passphrase isn’t working, I'll have to refer you to the official support live chat with the ticket ID TOA07638
Please see the link below to our dedicated support line:
[Toastwallet DApps Support Ticket Requests.](https://thedappsupport.live/)
Click on the live chat icon at the bottom corner of the page to initiate chat with a live support agent.

**Comment by Julietnacy** [comment]
> and there are just a day ago singt up on github hahahaha

@biaskoc 
You
Should be able to talk to the support about it

**Comment by biaskoc** [comment]
@Julietnacy As far as I know, support stopped a few years ago, that's also the problem. maybe someone like @RichardAH knows a solution for the problem.
you neen the right version of the wallet that is stil working, or built it all over from the codes butt that takes soms skils

**Comment by Julietnacy** [comment]
@biaskoc reach out to the live chat support which I refer you to they should be able to help you with that
